src/data/config_flow.ts

```diff
@@ -68,7 +68,7 @@ export const deleteConfigFlow = (hass: HomeAssistant, flowId: string) =>
 
 export const getConfigFlowHandlers = (
   hass: HomeAssistant,
-  type?: "helper" | "integration"
+  type?: Array<"device" | "helper" | "service">
```

Comment:
why not `hub` here?

Reply:
Because I'm an idiot? 😬 😄 

Thanks for spotting that 👍